sos/plugins/validations.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (C) 2020 Red Hat, Inc., Cedric Jeanneret <cjeanner@redhat.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.plugins import Plugin, RedHatPlugin
+import re
+
+
+# Notes:
+# - The Validation Framework is, for now, linked to openstack and tripleo
+# - Since the intent is to open it to other product (we can validate anything)
+#   this plugin has a generic name.
+# - The Framework is targeted at Red Hat products, at least for now.
+class Validations(Plugin, RedHatPlugin):
+    """Fetch run logs provided by the Validation Framework
+    """
+
+    plugin_name = 'validations'
```

Comment:
I'm not a huge fan of this plugin name. 'Validations', to me at least, makes it sound like sos is doing some, well, validation of the system by itself.

On the other hand, `validations_framework` is bit hefty for a plugin name - but `openstack_validations` isn't any better.

Reply:
I know :(. Also gave some thoughts when creating this patch, and, well... "let's push it like that and get feedbacks".
Unfortunately, we don't have another name for it, at least not official - I came with "valhalla[1]" as an acronym, but that's probably not better ;).

[1] Validate A Lot Hence Avoid Loads of Liabilities and Accounts

Reply:
@EmilienM any idea maybe?

Reply:
A few ideas:
* infra_validations
* cloud_validations

Or just "tripleo_validations"; as for now the validations are part of OpenStack TripleO governance.

Reply:
Thank you @EmilienM ! infra_validations might fit: generic enough to avoid renaming in the future, and not generic enough to cause issues.
@TurboTurtle would "infra_validations" be OK?

Reply:
I'd say I prefer the longer `validation_framework`, but I'm definitely open to further suggestions.

The plugin looks good to me right now pending a decision on the name. Let's leave this open for a little bit to get some more feedback/suggestions on that end.

Reply:
hello there! Is there anyone we could ping in order to get feedback about that name "issue"?

Reply:
We were discussing it internally. TL;DR the consensus is that the `validation_framework` is the best from "not great" names, while we dont have any great one.

Reply:
hello @pmoravec ! thanks for the update, I'll push that new name shortly then - at least there's some kind of consensus :)